### ISSUE TITLE
Buttons Touch target size

### ISSUE THREAD

**Issue by dmpr0** [osmandapp/OsmAnd#17174]
## ToDo
Default buttons height that we use is 36dp, Android is recommend to make touch target size 48dp.

- [x]  Button min. height should be `36dp`
- [x] Touch target size (green) `48dp`. Currently is the same 36dp.
- [x] Adjust paddings to visually balancing changes.

<table style="width:100%">
  <tr>
    <th></th>
    <th></th>
  </tr>
  <tr>
    <td><img src="https://github.com/osmandapp/OsmAnd/assets/9365308/9ae211f2-3783-4428-ab6e-2ff77cd3727a"  width="300" height="auto"/> </td>
    <td><img src="https://github.com/osmandapp/OsmAnd/assets/9365308/2d43d235-1cb4-4223-a8ad-3072365ec55b"  width="300" height="auto"/> </td>
  </tr>
  <tr>
    <th></th>
    <th></th>
  </tr>
  <tr>
    <td><img src="https://github.com/osmandapp/OsmAnd/assets/9365308/080ce042-68a7-43fd-9909-fe20ecb51428"  width="300" height="auto"/> </td>
    <td><img src="https://github.com/osmandapp/OsmAnd/assets/9365308/f2e842cd-d818-4378-ac17-4fca58074885"  width="300" height="auto"/> </td>
  </tr>
</table>

**Comment by dmpr0** [comment]
# Review
- [ ]  OsmAnd Cloud after login
<table style="width:100%">
  <tr>
    <td><img src="https://github.com/osmandapp/OsmAnd/assets/9365308/e4c0b4ce-54a9-4b50-aa25-98c197c2208d"  width="300" height="auto"/> </td>
  </tr>
</table>
